docs(redis) redis.username cannot be 'default' (#4913)
Kong/docs.konghq.com: app/_hub/kong-inc/rate-limiting-advanced/_index.md

      referenceable: true
      description: |
        Username to use for Redis connection when the `redis` strategy is defined and ACL authentication is desired.

>         If undefined, ACL authentication will not be performed. This requires Redis v6.0.0+. Note that you can not set it as *default*.

Reviewer: ```suggestion
        If undefined, ACL authentication will not be performed. This requires Redis v6.0.0+. This can not be set to **default**.
```
